src/utils/calculateTooltipOffset.test.js

```diff
@@ -2,14 +2,14 @@ import calculateTooltipOffset from './calculateTooltipOffset';
 
 describe('Tooltip offset calculator', () => {
   it('should calculate offset for a 2 hour meeting', () => {
-    expect(calculateTooltipOffset(2)).toBe(-54);
+    expect(calculateTooltipOffset(2)).toBe(-54.5);
```

Comment:
If you will notice, every offset is now smaller by .5, which is the extra 1px divided by two.